clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemUtils.java

```diff
@@ -134,4 +142,42 @@ public static boolean waitCompleted(final TachyonFileSystemCore tfs, final Tachy
 
     return completed;
   }
+
+  /**
+   * Persist the given file to the under file system.
+   *
+   * @param tfs {@link TachyonFileSystem} to carry out tachyon operations
+   * @param file the file to persist
+   * @param fileInfo the file info of the file
+   * @param tachyonConf TachyonConf object
+   * @return the size of the file persisted
+   * @throws IOException if an I/O error occurs
+   * @throws FileDoesNotExistException if the given file does not exist
+   * @throws TachyonException if an unexpected Tachyon error occurs
+   */
+  public static long persistFile(TachyonFileSystem tfs, TachyonFile file, FileInfo fileInfo,
```

Comment:
@manugoyal Could we just have the redundant rpc call instead of passing FileInfo? In the future we may merge `FileInfo` into `TachyonFile` which may solve this problem. Also, I think we can remove TachyonConf and just use `ClientContext.getConf` since this will not be used by any server side code.